src/features/views/utils/getUniqueColumnName.ts

```diff
@@ -0,0 +1,20 @@
+import { ZetkinViewColumn } from '../components/types';
+
+export default function getUniqueColumnName(
+  defaultTitle: string,
+  columns: ZetkinViewColumn[]
+) {
+  let title = defaultTitle;
+  let count = 2;
+  let foundNewTitle = false;
+  while (!foundNewTitle) {
```

Comment:
I would replace this with `while (true)` and then `break` out of the loop when the first unique title is found (on line 15). That way, you can remove the variable.